src/tests/Interop/COM/NativeServer/ErrorMarshalTesting.h

```diff
@@ -26,12 +26,38 @@ class ErrorMarshalTesting : public UnknownImpl, public IErrorMarshalTesting
         return hresultToReturn;
     }
 
+    DEF_FUNC(Throw_HResult_HelpLink)(
+        /*[in]*/ int hresultToReturn,
+        /*[in]*/ LPCWSTR helpLink,
+        /*[in]*/ DWORD helpContext)
+    {
+        HRESULT hr;
+
+        ComSmartPtr<ICreateErrorInfo> pCreateErrInfo;
+        BSTR bstrHelpLink = SysAllocString(helpLink);
+        RETURN_IF_FAILED(::CreateErrorInfo(&pCreateErrInfo));
+        RETURN_IF_FAILED(pCreateErrInfo->SetHelpFile(bstrHelpLink));
+        RETURN_IF_FAILED(pCreateErrInfo->SetHelpContext(helpContext));
+
+        ComSmartPtr<IErrorInfo> pErrInfo;
+        RETURN_IF_FAILED(pCreateErrInfo->QueryInterface(IID_IErrorInfo, (void**)&pErrInfo));
+        RETURN_IF_FAILED(SetErrorInfo(0, pErrInfo));
+
+        return HRESULT{ hresultToReturn };
+    }
+
+    DEF_FUNC(InterfaceSupportsErrorInfo)(
+        /* [in] */ __RPC__in REFIID riid)
+    {
+        return S_OK;
```

Comment:
This should only return `S_OK` when the interface queried supports it. In this case, I think that is `IErrorMarshalTesting`.